demo/attester/attester.Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ubuntu:20.04
 
+ARG TARGETARCH
```

Comment:
Hmm, this was already fixed in [my PR](https://github.com/CCC-Attestation/attested-tls-poc/pull/46/files). I get the feeling that our merge strategy isn't ideal :(

Reply:
rebase done